### PR TITLE
sqlcheck: init at 1.2

### PR DESCRIPTION
**Pull request by marsam**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [X] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -1676,6 +1676,8 @@ with pkgs;
 
   sqlint = callPackage ../development/tools/sqlint { };
 
+  sqlcheck = callPackage ../development/tools/sqlcheck { };
```
Comment (c0bw3b) [inline pkgs/top-level/all-packages.nix]: It should be further down in the `### DEVELOPMENT / TOOLS` section, right before `sqlitebrowser`

Also your default.nix could live under `development/tools/database/sqlcheck/` bat that's a minor nitpick.

Comment (marsam) [reply pkgs/top-level/all-packages.nix]: sure, updated

**Comment by c0bw3b** [comment]
@GrahamcOfBorg build sqlcheck

**Comment by c0bw3b** [comment]
Thanks

**Pull request merged**